src/nvim/lua/treesitter.c

```diff
@@ -1310,9 +1319,10 @@ static void push_querymatch(lua_State *L, TSQueryMatch *match, int uindex)
 
 static TSQueryMatch *querymatch_check(lua_State *L, int index)
 {
-  TSQueryMatch *ud = luaL_checkudata(L, 1, TS_META_QUERYMATCH);
+  TSQueryMatch *ud = luaL_checkudata(L, index, TS_META_QUERYMATCH);
```

Comment:
typo fix

Reply:
This fix is enough to prevent the `format-queries` error.